.circleci/scripts/bundle-stats-commit.sh

```diff
@@ -16,6 +16,12 @@ then
     exit 1
 fi
 
+if [[ "${CIRCLE_BRANCH}" != "develop" ]]
+then
+    printf 'This is not develop branch'
+    exit 0
+fi
+
```

Comment:
I have done this additional change in PR to move check for branch before check for `GITHUB_TOKEN` so that the check is only done on develop branch.